src/hypothesis/internal/compat.py

```diff
@@ -392,7 +392,7 @@ def __add__(self, value):
         return compatbytes(bytearray.__add__(self, value))
 
     def __radd__(self, value):
-        return compatbytes(bytearray.__radd__(self, value))
+        return compatbytes(bytearray.__add__(value, self))
```

Comment:
If it was never invoked in normal code, how did it get through the coverage checks? Why did we never spot the non-existence of `bytearray.__radd__`?